youtube_dl/extractor/nrk.py

```diff
@@ -105,6 +106,7 @@ def video_id_and_title(idx):
             MESSAGES = {
                 'ProgramRightsAreNotReady': 'Du kan dessverre ikke se eller høre programmet',
                 'ProgramRightsHasExpired': 'Programmet har gått ut',
+                'NoProgramRights': 'Ikke tilgjengelig',
```

Comment:
Provide example URL.

Reply:
Do you want it in the code, or in this discussion?

Reply:
Add as a test.

Reply:
A valid url for the message is already in a test, second on NRKTV